src/client/commands/wait.rs

```diff
@@ -0,0 +1,68 @@
+use crate::JobId;
+
+use crate::rpc_call;
+use clap::Clap;
+
+use crate::client::globalsettings::GlobalSettings;
+use crate::client::status::is_terminated;
+use crate::server::bootstrap::get_client_connection;
+use crate::transfer::messages::{
+    FromClientMessage, JobInfo, JobInfoRequest, JobSelector, ToClientMessage,
+};
+use std::thread::sleep;
+use std::time::Duration;
+
+#[derive(Clap, Copy, Clone)]
+#[clap(setting = clap::AppSettings::ColoredHelp)]
+pub struct WaitOpts {
+    job_id: JobId,
+}
+
+pub async fn wait_on_job(gsettings: &GlobalSettings, opts: WaitOpts) -> anyhow::Result<()> {
+    let mut response = rpc_call!(
+        get_client_connection(&gsettings.server_directory()).await?,
+        FromClientMessage::JobInfo(JobInfoRequest {
+            selector: JobSelector::Specific(vec![opts.job_id]),
+        }),
+        ToClientMessage::JobInfoResponse(r) => r
+    )
+    .await?;
+    let mut info: &JobInfo = response.jobs.get(0).unwrap();
```

Comment:
First you should check if it's empty, otherwise the unwrap could crash the program.